sympy/assumptions/assume.py

```diff
@@ -50,27 +79,44 @@ def _sympystr(self, printer):
 
 
 class AppliedPredicate(Boolean):
-    """The class of expressions resulting from applying a Predicate.
+    """
+    The class of expressions resulting from applying ``Predicate`` to
+    the arguments. ``AppliedPredicate`` merely wraps its argument and
+    remain unevaluated. To evaluate it, use ``ask`` function.
 
     Examples
     ========
 
-    >>> from sympy import Q, Symbol
-    >>> x = Symbol('x')
-    >>> Q.integer(x)
-    Q.integer(x)
-    >>> type(Q.integer(x))
+    >>> from sympy import Q, ask
+    >>> Q.integer(1)
+    Q.integer(1)
+
+    ``function`` attribute returns the predicate, and ``arguments``
```

Comment:
```suggestion
    The ``function`` attribute returns the predicate, and the ``arguments``
```

Reply:
This sentence would be good in the docstring for `function` and `arguments` too,